prow/ANNOUNCEMENTS.md

```diff
@@ -2,6 +2,7 @@
 
 New features added to each component:
 
+ - *February 13, 2019* `horologium` and `sinker` deployments will soon require `--dry-run=false` in production, please set this before March 15.
```

Comment:
do we need to remove the previous announcement? ( line 48 ) - and technically this is not a "new feature" but more like a future breaking change?

Reply:
Done